app/src/main/java/org/mozilla/focus/home/pocket/PocketVideoFragment.kt

```diff
@@ -86,6 +96,11 @@ class PocketVideoFragment : Fragment() {
         }
     }
 
+    override fun onStop() {
+        super.onStop()
+        uiLifecycleCancelJob.cancel()
```

Comment:
After this point, this job is perpetually cancelled and we never attach a new job to it. I think you should be creating/cancelling new parent jobs in onStart/onStop or `onCreateView/onDestroyView`.